portfolio/src/main/java/com/google/sps/util/SentimentAnalysisUtil.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.util;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+import java.util.Optional;
+
+public class SentimentAnalysisUtil {
+  private final Optional<LanguageServiceClient> languageService;
```

Comment:
Final variable since we only instantiate it once inside the constructor; non-static due to possibility of IOException being thrown during instantiation